Source/Core/Resource.js

```diff
@@ -1209,6 +1260,38 @@ define([
         }
     }
 
+    /**
+     * Asynchronously loads the given resource.  Returns a promise that will resolve to
+     * the result once loaded, or reject if the resource failed to load.  The data is loaded
+     * using XMLHttpRequest, which means that in order to make requests to another origin,
+     * the server must have Cross-Origin Resource Sharing (CORS) headers enabled.
+     *
+     * @param {Object} [options] Object with the following properties:
+     * @param {String} [options.responseType] The type of response.  This controls the type of item returned.
+     * @param {Object} [options.headers] Additional HTTP headers to send with the request, if any.
+     * @param {String} [options.overrideMimeType] Overrides the MIME type returned by the server.
+     * @returns {Promise.<Object>|undefined} a promise that will resolve to the requested data when loaded. Returns undefined if <code>request.throttle</code> is true and the request does not have high enough priority.
+     *
+     *
+     * @example
+     * // Load a single resource asynchronously. In real code, you should use loadBlob instead.
```

Comment:
What does `In real code, you should use loadBlob instead` mean?  Is that just a bad copy/paste?  It's in the doc for the next few functions too

Reply:
I think the answer is "both" 😄 

Ge means that you wouldn't normally use `fetch` to retrieve a blob, you would call the more specific `fetchBlob` instead.

@tfili I would recommend deleting this comment and updating the comment paragraph above to say that we recommend using the more specific functions in the class, i.e. fetchJson, fetchImage, etc.. rather than calling fetch directly, but we expose this for the cases where the user does not want the data to be interested in anyway way.  Then just rename blob in the below example to `body` and we're good to go.